noxfile.py

```diff
@@ -221,6 +222,63 @@ def upload_wheel(session: nox.Session) -> None:
     session.run("twine", "upload", "dist/*", external=True)
 
 
+@nox.session
+def bundle_app(session: nox.Session) -> None:
+    """Bundle a standalone executable."""
+    session.install(
+        "jinja2",
+        "lxml",
+        "pygments",
+        "pyinstaller",
+    )
+    session.install("-e", ".")
+    if not os.path.exists("build"):
+        os.mkdir("build")
+    session.chdir("build")
+    main_py = "main.py"
+    with open(main_py, "w") as FH:
+        FH.write(
+            textwrap.dedent(
+                """\
+                import sys
+                import runpy
+
+                if __name__ == "__main__":
+                    runpy.run_module("gcovr", run_name="__main__", alter_sys=True)
+                    sys.exit(0)
+                """
+            )
+        )
```

Comment:
If we're creating a temporary file like this, we might as well add it directly as `scripts/pyinstaller_entrypoint.py` if that works.

Reply:
This I need to do after my holiday. That's impossible on my mobile phone 😄

Reply:
If we use the wheel and there is an easy way to get the path to gcovr (independent of installation type global, local or venv) this will also work.

Reply:
@latk I've managed to use the script generated by setuptool. For Windows I need to find a way to extract the script from the executable.